conversation_reconstruction_dataflow/conversation_reconstruction/mwparserfromhell.egg-info/PKG-INFO

```diff
@@ -0,0 +1,11 @@
+Metadata-Version: 1.0
```

Comment:
If we're including wikimedia parser, it should live in a subdirectory of a directory called "third_party/" (same for all included third party libraries)

Reply:
Should there be extra documentation about third party packages?